go.mod

```diff
@@ -42,7 +42,7 @@ require (
 	github.com/moby/sys/signal v0.6.0
 	github.com/moby/sys/symlink v0.2.0
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/opencontainers/image-spec v1.0.2-0.20211117181255-693428a734f5
+	github.com/opencontainers/image-spec v1.0.2 // see replace for the actual version
```

Comment:
this should be https://github.com/moby/buildkit/pull/2764/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R42

Reply:
Ah, right, so that's the merge commit that merged back the v1.0 branch into main, right? https://github.com/opencontainers/image-spec/compare/693428a734f5...c5a74bcca799

I guess that fixes it, without introducing changes, yes (thanks!)

It's still somewhat ridiculous that these things are needed (especially the "roll back to what we _think_ is more current).

Reply:
opened https://github.com/containerd/containerd/pull/6766